tests/unittests/utils/configuration/test_settings.py

```diff
@@ -158,3 +157,14 @@ def test_attributes_whose_key_is_absent_return_expected_default(
         config = UDCTokenSettings(minimal_yaml_dict)
         MISSING = object()
         assert getattr(config, key, MISSING) == expected
+
+    def test_balance_per_node_bigger_than_max_funding(self, minimal_yaml_dict):
+        minimal_yaml_dict["max_funding"] = 5000
+        minimal_yaml_dict["balance_per_node"] = 5001
+        assert UDCTokenSettings(minimal_yaml_dict).CONFIGURATION_ERROR == ConfigurationError
+
+    # FIXME this test doesn't work for whatever reason
+    def test_insufficient_minting(self, minimal_yaml_dict):
+        minimal_yaml_dict["max_funding"] = 5000
+        minimal_yaml_dict["token"]["min_balance"] = 4999
+        assert UDCTokenSettings(minimal_yaml_dict).CONFIGURATION_ERROR == ConfigurationError
```

Comment:
I think there's a misconception about that `.CONFIGURATION_ERROR` is supposed to be:
That's the exception type to be invoked by `.validate` - it does *not* store exceptions raised during `validate()`. 

Instead, this should be:

```suggestion
with pytest.raises(ConfigurationError):
    UDCTokenSettings(minimal_yaml_dict)
```

It would also be reasonable to create a `UDCTokenConfigError`, and assign that to the class's `CONFIGURATION_ERROR` attribute, and change `pytest.raises` to look for that instead.

Reply:
Yep, it was a misconception. Changed it.